### PR TITLE
Set Cassandra default replication factor to 3

### PR DESCRIPTION
**Pull request by Torch3333**
[As discussed](https://github.com/scalar-labs/scalardb/pull/989#discussion_r1285579493) in a previous PR, this updates the default replication factor of namespace created via the Cassandra Admin and Schema Loader from 1 to 3.

### CODE REVIEW

File: core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java [hunk 1]
```diff
@@ -86,7 +87,7 @@ private void createKeyspace(String keyspace, Map<String, String> options, boolea
     if (ifNotExists) {
       query = query.ifNotExists();
     }
-    String replicationFactor = options.getOrDefault(REPLICATION_FACTOR, "1");
+    String replicationFactor = options.getOrDefault(REPLICATION_FACTOR, DEFAULT_REPLICATION_FACTOR);
```
Comment (Torch3333) [inline core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java:90]: Updates the default replication factor from 1 to 3.

File: docs/getting-started-with-scalardb-by-using-kotlin.md [hunk 2]
```diff
@@ -198,7 +198,7 @@ To apply the schema, go to the [`scalardb` Releases](https://github.com/scalar-l
 Then, run the following command, replacing `<VERSION>` with the version of the ScalarDB Schema Loader that you downloaded:
 
 ```shell
-$ java -jar scalardb-schema-loader-<VERSION>.jar --config scalardb.properties --schema-file emoney.json --coordinator
+$ java -jar scalardb-schema-loader-<VERSION>.jar --config scalardb.properties --schema-file emoney.json --coordinator --replication-factor=1
```
Comment (Torch3333) [inline docs/getting-started-with-scalardb-by-using-kotlin.md:201]: @josh-wong Please review the changes in Java and Kotlin getting started documents when you can.

Ideally, we could display the `--replication-factor=1` option and the note below only when the user has selected Cassandra as the storage, as we do for the previous step, but I am unsure of the best way to do this and if it is worth it.

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by komamitsu — APPROVED** [review]
LGTM! Thank you!

File: docs/getting-started-with-scalardb-by-using-kotlin.md (hunk 2 above)
Comment (josh-wong) [reply docs/getting-started-with-scalardb-by-using-kotlin.md:201]: @Torch3333 How about if we use content tabs to show the commands and notice block for each database?

Attached is a video of what that would look like.

https://github.com/scalar-labs/scalardb/assets/23216828/e610fd9c-76db-495d-8bf7-b82729a160da

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

File: docs/getting-started-with-scalardb-by-using-kotlin.md (hunk 2 above)
Comment (Torch3333) [reply docs/getting-started-with-scalardb-by-using-kotlin.md:201]: That looks good, thank you. Let's go with your suggestion.

Comment (Torch3333) [reply docs/getting-started-with-scalardb-by-using-kotlin.md:201]: I can modify both readme files if you suggest to me how to and you could push changes directly to this PR. Whichever is fine.

Comment (josh-wong) [reply docs/getting-started-with-scalardb-by-using-kotlin.md:211]: I've added the code for the content tabs (tested in my local environment) and slightly edited the new content😊

Comment (Torch3333) [reply docs/getting-started-with-scalardb-by-using-kotlin.md:211]: Great, thank you.

**Review by josh-wong — APPROVED** [review]
LGTM! Thank you!

> **Note**
> 
> I added content tabs as mentioned in a [comment reply](https://github.com/scalar-labs/scalardb/pull/1080/files#r1323863380).

**Pull request merged**